app/models/invenio_rdm_record_converter.rb

```diff
@@ -181,7 +181,7 @@ def invenio_metadata
       "subjects": SUBJECT_SCHEMES.map{ |subject_type| subjects_for_scheme(@generic_file.send(subject_type), subject_type) }.compact.flatten,
       "contributors": contributors(@generic_file.contributor),
       "dates": @generic_file.date_created.map{ |date| {"date": date, "type": "other", "description": "When the item was originally created."} },
-      "languages": @generic_file.language.map{ |lang| lang.present? && lang.downcase == ENGLISH ? {"id": "eng"} : nil }.compact,
+      "languages": @generic_file.language.map{ |lang| lang.present? && lang.downcase == ENGLISH ? "eng" : nil }.compact,
```

Comment:
Totally my fault, the documentation was incorrect :disappointed: . I fixed it. This can be left as it was before i.e. `{"id": "eng"}`.